extensions/localserver-cli-command/src/DeployXml.js

```diff
@@ -16,12 +16,20 @@ module.exports = class DeployXml {
 
 		this.deploy = Utils.parseXml(this._projectFolder, 'deploy.xml').deploy;
 
+		const { manifest } = Utils.parseXml(this._projectFolder, 'manifest.xml');
```

Comment:
We have constants for manifest.xml and ACCONTCUSTOMIZATION in ApplicationConstants.js
I would use those here and we can add another constrant for 'deploy.xml' under FILES